lib/drivers/medtronic600/NGPHistoryParser.js

```diff
@@ -798,10 +826,23 @@ class NGPHistoryParser {
   }
 
   processPages(pages) {
+    let skipTimeChange = false;
     // Consume the pages as we process them to save on memory.
     while (pages.length > 0) {
       const page = pages.shift();
       for (const event of NGPHistoryParser.eventsFromPageString(page)) {
+        if (event.eventType === NGPHistoryEvent.EVENT_TYPE.TIME_RESET) {
+          debug('TIME_RESET detected. Ignoring next USER_TIME_DATE_CHANGE.');
```

Comment:
Do we also need to ignore other events/alarms between `TIME_RESET` and `USER_TIME_DATE_CHANGE`? Or do we want to still record them even if their datetime is incorrect? Maybe we should annotate them?

Reply:
There are other events in there that we don't process currently (DAILY_TOTALS, other ALARMS). Currently I'm effectively ignoring them by not processing them. Perhaps I should explicitly ignore them by not pushing them. Thoughts?

Reply:
Let's make that decision when we start to process them.